releases/UWP/net-native2.0/README.md

```diff
@@ -3,6 +3,9 @@
 You can see what was included in each .NET Native 2.0 ([Microsoft.NETCore.UniversalWindowsPlatform 6.0.x](https://www.nuget.org/packages/Microsoft.NETCore.UniversalWindowsPlatform)) release below.
 
 When using Visual Studio these packages require Visual Studio 2017 Version 15.4 or later.
+### UWP 6.0.7 (.NET native tools 2.0.2) (February 7th, 2018)
+- ILC updated to copy all .pdb files to the output .appxsyms, including ones that do not have matching binaries. #524755
```

Comment:
Nit: The previous notes used [524755] format for the bug numbers.

Reply:
Perhaps you could reword the beginning to read "Updated the compiler to copy...". Let me know what you think of the suggestion.